lib/hammer_cli_foreman/audit.rb

```diff
@@ -0,0 +1,36 @@
+module HammerCLIForeman
+
+  class Audit < HammerCLIForeman::Command
+
+    resource :audits
+
+    class ListCommand < HammerCLIForeman::ListCommand
+
+      output do
+        field :id, _("Id")
+        field :created_at, _("At")
+        field :remote_address, _("IP")
+        field :user_name, _("User name")
+        field :action, _("Action")
+        field :auditable_type, _("Audit type")
+        field :auditable_name, _("Audit name")
```

Comment:
Similar here:
```ruby
field nil, _("Audited record"), Fields::SingleReference, :key => :auditable
```
I'm not sure about what's the best label for this field though.